thirdeye/thirdeye-frontend/app/pods/home/share-dashboard/route.js

```diff
@@ -97,17 +104,25 @@ export default Route.extend(AuthenticatedRouteMixin, {
 
   _getAnomalyMapping: task (function * () {//TODO: need to add to anomaly util - LH
     let anomalyMapping = {};
+    let anomalies;
     //fetch the anomalies from the onion wrapper cache.
-    const applicationAnomalies = yield get(this, 'anomaliesApiService').queryAnomaliesByAppName(get(this, 'appName'), get(this, 'startDate'), get(this, 'endDate'));
+    if (this.get('appName') && this.get('subGroup')) {
+      //this functionality is not provided in the UI, but the user can manually type the params into URL simultaneously
+      anomalies = yield this.get('anomaliesApiService').queryAnomaliesByJoin(this.get('appName'), this.get('subGroup'), this.get('startDate'), this.get('endDate'));
+    } else if (this.get('appName')) {
+      anomalies = yield this.get('anomaliesApiService').queryAnomaliesByAppName(this.get('appName'), this.get('startDate'), this.get('endDate'));
+    } else {
+      anomalies = yield this.get('anomaliesApiService').queryAnomaliesBySubGroup(this.get('subGroup'), this.get('startDate'), this.get('endDate'));
```

Comment:
Which endpoint is this calling? So that I can double check the backend to see if it's correct.

Reply:
Here's an example, with appName and group seeming to have join behavior, based on the responses I've gotten:

/userdashboard/anomalies?appName={someAppName}group={someSubGroup}

Reply:
Right. Doubled check this will work.